### ISSUE TITLE
Deduce size of output logits in handler

### ISSUE THREAD

**Issue by johanos1** [aidotse/LeakPro#176]
# Feature
## Desired Behavior / Functionality
Right now, each attack is responsible for finding out the size of the logits.

## What Needs to Be Done
This should be done once only in the handler.

## How Can It Be Tested
Attacks work for both single neuron output and multiple.
